### PR TITLE
Head fixes

### PR DESCRIPTION
**Pull request by cbosdo**
## What does this PR change?

**add description**

### CODE REVIEW

**Review by srbarrios — APPROVED** [review]

File: salt/repos/server_containerizedUyuni.sls [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+systemsmanagement_Uyuni_Master_ContainerUtils:
+    pkgrepo.managed:
+    - baseurl: http://{{ grains.get("mirror") | default("downloadcontent.opensuse.org", true) }}/repositories/systemsmanagement:/Uyuni:/Master:/ContainerUtils/{{ repo }}/
+    - refresh: True
+    - gpgkey: http://{{ grains.get("mirror") | default("downloadcontent.opensuse.org", true) }}/repositories/systemsmanagement:/Uyuni:/Master:/ContainerUtils/{{ repo }}/repodata/repomd.xml.key
+
+{% endif %}
+
+
+# WORKAROUND: see github:saltstack/salt#10852
```
Comment (nodeg) [inline salt/repos/server_containerizedUyuni.sls:21]: This issue was already closed and referenced a new one [here](https://github.com/saltstack/salt/issues/65080). However, the new issue was marked as resolved/fixed.

Comment (nodeg) [reply salt/repos/server_containerizedUyuni.sls:21]: But I guess our Salt version is older and does not include this fix.

File: salt/repos/server_containerized.sls [hunk 2]
```diff
@@ -1,10 +1,11 @@
 {% if 'server_containerized' in grains.get('roles')  %}
 
-systemsmanagement_Uyuni_Master_ContainerUtils:
-    pkgrepo.managed:
-    - baseurl: http://{{ grains.get("mirror") | default("downloadcontent.opensuse.org", true) }}/repositories/systemsmanagement:/Uyuni:/Master:/ContainerUtils/openSUSE_Leap_15.5/
-    - refresh: True
-    - gpgkey: http://{{ grains.get("mirror") | default("downloadcontent.opensuse.org", true) }}/repositories/systemsmanagement:/Uyuni:/Master:/ContainerUtils/openSUSE_Leap_15.5/repodata/repomd.xml.key
+include:
+  {%- if 'head' in grains['product_version'] %}
```
Comment (nodeg) [inline salt/repos/server_containerized.sls:4]: I think for this to work, we need to adjust the containerized server tests.

Comment (nodeg) [reply salt/repos/server_containerized.sls:4]: https://github.com/uyuni-project/sumaform/pull/1481

File: salt/repos/server_containerizedUyuni.sls [hunk 3]
```diff
@@ -0,0 +1,24 @@
+{% if 'server_containerized' in grains.get('roles')  %}
+
+{% if grains.get("os") == "SUSE" %}
+{% if grains['osfullname'] == 'Leap' %}
+{% set repo = 'openSUSE_Leap_15.5' }
```
Comment (srbarrios) [inline salt/repos/server_containerizedUyuni.sls:5]:
```suggestion
{% set repo = 'openSUSE_Leap_15.5' %}
```

File: salt/repos/server_containerizedUyuni.sls [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+{% elif grains['osfullname'].startswith('SLE') %}
+{% set repo = 'SLE_15' }
```
Comment (srbarrios) [inline salt/repos/server_containerizedUyuni.sls:7]:
```suggestion
{% set repo = 'SLE_15' %}
```

File: salt/repos/server_containerizedUyuni.sls [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+{% else %}
+{% set repo = 'openSUSE_Leap_Micro_5.5' }
```
Comment (srbarrios) [inline salt/repos/server_containerizedUyuni.sls:9]:
```suggestion
{% set repo = 'openSUSE_Leap_Micro_5.5' %}
```

File: salt/repos/server_containerizedUyuni.sls (hunk 1 above)
Comment (cbosdo) [reply salt/repos/server_containerizedUyuni.sls:21]: I confirm we need it for requires

File: salt/repos/server_containerized.sls (hunk 2 above)
Comment (cbosdo) [reply salt/repos/server_containerized.sls:4]: Sure container tests for Head will need to use `head`

**Comment by cbosdo** [comment]
This PR requires https://build.opensuse.org/request/show/1141772

File: backend_modules/libvirt/base/main.tf [hunk 6]
```diff
@@ -46,6 +46,7 @@ locals {
     slemicro53-ign           = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.opensuse.org"}/repositories/systemsmanagement:/sumaform:/images:/microos/images_53/SLE-Micro.x86_64-sumaform.qcow2"
     slemicro54-ign           = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.opensuse.org"}/repositories/systemsmanagement:/sumaform:/images:/microos/images_54/SLE-Micro.x86_64-sumaform.qcow2"
     slemicro55-ign           = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.opensuse.org"}/repositories/systemsmanagement:/sumaform:/images:/microos/images_55/SLE-Micro.x86_64-sumaform.qcow2"
+    slemicro55o              = "${var.use_mirror_images ? "http://${var.mirror}" : "http://download.opensuse.org"}/repositories/systemsmanagement:/sumaform:/images:/microos/images_55/SLE-Micro.x86_64-sumaform.qcow2"
```
Comment (srbarrios) [inline backend_modules/libvirt/base/main.tf:49]: This URL is the same we use in `slemicro55-ign`

**Pull request merged**